src/lambdas/inspect/audio.js

```diff
@@ -21,6 +23,7 @@ import { nmbr } from "./util.js";
  * @property {number} [LoudnessTruePeak]
  * @property {number} [LoudnessRange]
  * @property {number} [UnidentifiedBytes]
+ * @property {Tag[]} [Tags]
```

Comment:
This  used to be a hash, but is now a list of objects, as the same key may be used for more than one tag.